src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -1,6 +1,15 @@
+import _ from 'lodash/fp';
 import { transformForSubmit } from '../../common/schemaform/helpers';
 
 export function transform(formConfig, form) {
+  // Flatten claimedSponsorService; it was grouped like this to get expandUnder
+  //  working properly, but now we need to flatten and remove it before sending
+  //  to the api.
+  const modifiedForm = _.set('benefitSelection.data.previousBenefits', {
+    ...form.benefitSelection.data.previousBenefits.claimedSponsorService
+  }, form);
+  delete modifiedForm.benefitSelection.data.previousBenefits.claimedSponsorService;
```

Comment:
For now, I only _assume_ this works; I don't know how to test it until we get the whole form up and running.